include/stumpless/error.h

```diff
@@ -156,7 +156,13 @@ extern "C" {
  *
  * @since release v2.2.0
  */\
-  ERROR( STUMPLESS_NETWORK_CLOSED, 33 )
+  ERROR( STUMPLESS_NETWORK_CLOSED, 33 ) \
+/** 
+ * A provided argument was too big, for example maximum alloted buffer size. 
```

Comment:
This description should be fixed to describe the provided argument as too small.